templates/public/feeds.html

```diff
@@ -29,7 +29,7 @@
     Alternatively, you can select a packages feed from the below table that is
```

Comment:
I'd rather just remove entire block about architecture specific feeds given we will have only one for now.

Reply:
Agreed

Reply:
Hmm well we will still show "any" / "x86_64" and we will still show "All arches". I believe we should keep this since in the future we might want to support a new Arch.